README.md

```diff
@@ -1,2 +1,35 @@
 # ex_limiter
 Rate Limiter written in elixir with configurable backends
+
+Implements leaky bucket rate limiting, which is superior to most naive approaches by handling bursts even around time windows.  You can define your own storage backend by implementing the `ExLimiter.Storage` behaviour, and configuring it with
```

Comment:
Maybe it's worth providing a wiki link or something https://en.wikipedia.org/wiki/Leaky_bucket